pom.xml

```diff
@@ -93,6 +93,7 @@
     <google.auto-value.version>1.8.2</google.auto-value.version>
     <google.cloud-bigquerystorage.version>2.25.0</google.cloud-bigquerystorage.version>
     <google.cloud-core.version>2.5.4</google.cloud-core.version>
+    <google.cloud-storage.bom.version>2.15.0</google.cloud-storage.bom.version>
     <google.cloud-storage.grpc.version>2.2.2-alpha</google.cloud-storage.grpc.version>
```

Comment:
Is this property used anywhere after this change?

Reply:
It is getting used to pick the version for `google-cloud-storage-bom` artifact.
We are integrating new java-client created by gcs team.

Reply:
I meant `google.cloud-storage.grpc.version` property. Seems we removed dependency that used it directly, so we can remove this property too?

Reply:
aah. Yeah It can be removed.